assets/cms/scss/_panels.scss

```diff
@@ -1 +1,2 @@
 @import "panels/shared";
+//@import "panels/base";
```

Comment:
Do you mean to have this commented out?

Reply:
Yes, that's correct, we can remove it tho.

Reply:
This was corrected in the latest pull request.